Frontend/src/pages/dashboard/components/Farmer/StatisticsDashboard.jsx

```
@@ -7,88 +8,141 @@ import {
+    cropDistribution: [],
+  });
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchOrders = async () => {
+      try {
+        const response = await fetch("http://localhost:3000/api/orders"); // Adjust URL as needed
+        const orders = await response.json();
+
+        // Process data
+        const totalRevenue = orders.reduce((sum, order) => sum + parseFloat(order.price || 0), 0);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)